avocado/plugins/teststmpdir.py

```diff
@@ -0,0 +1,26 @@
+import os
+import shutil
+import tempfile
+
+from avocado.core.plugin_interfaces import JobPre, JobPost
+
+
+class TestsTmpDir(JobPre, JobPost):
+
+    name = 'teststmpdir'
+    description = 'Creates a temporary directory for tests consumption'
+
+    def __init__(self):
+        self._varname = 'XXX_TESTS_COMMON_TMPDIR'
```

Comment:
Well the `XXX_` prefix comes from assumption it'd be implemented as a contrib script, which is not in this version. So I'd go with `AVOCADO_` prefix, or moving this to contrib. I understand that having it as a contrib script is tempting, because we'd not have to guarantee anything, but at the same time I'd be hesitant to use it in `avocado-vt` if it's not enabled by default. So it's a hard cause and I think it's better having it implemented here, use the `AVOCADO_` name and eventually adopt it in `avocado-vt` is the way to go.